File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) 
    {
        if(x<0) return false;
        long rev=0;
        int temp=x;
        while(x!=0){
            int lastdigit=x%10;
            rev=(rev*10)+lastdigit;
            x=x/10;
        }
        return (rev==temp);
        
    }    
    
};

File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        numstr=str(x)
        rev_string=numstr[ : :-1]
        if(numstr==rev_string):
            return True
        else:
            return False
        